packages/scripts/webapp/scripts/build.js

```diff
@@ -1,23 +1,12 @@
 const spawn = require('cross-spawn');
 const { hereRelative, resolveBin } = require('../utils/path-resolver');
 
-const rimraf = resolveBin('rimraf');
 const webpack = resolveBin('webpack');
 
-module.exports = function build(env) {
-	// remove old builds folder
```

Comment:
We don't need to do this anymore?

Reply:
Sorry I mixed 2 PRs, while you were writing that, I was reverting it :)
This will come after, the target forlder removal will come from a webpack plugin instead, more maintainable because it takes the real target folder instead of duplicating it here